### DISCUSSION TITLE
Sorry, you have been blocked

### DISCUSSION THREAD

**Discussion by suwjie** [ultrafunkamsterdam/undetected-chromedriver#2095] [Q&A]
![image](https://github.com/user-attachments/assets/4f43fc78-afb0-4ae5-99b6-cc91a9a89e3b)
![image](https://github.com/user-attachments/assets/5e21d1cf-3c4f-4237-9fcd-c0a7c445e2ba)
![image](https://github.com/user-attachments/assets/89722631-9b7f-4694-9f81-fa9da44aa30b)
![image](https://github.com/user-attachments/assets/759f6b9d-ece9-4ef7-b72d-4f10fab84aae)
还是无法屏蔽cloudflare的识别！

**Comment by Eboubaker** [comment]
same issue here.

**Comment by Eboubaker** [comment]
@suwjie 如果你解决了这个问题，可以告诉我你是怎么解决的吗？

**Comment by ultrafunkamsterdam** [comment]
Using proxy, random user agent. And then complain here...
